images/win/scripts/Installers/Validate-Boost.ps1

```diff
@@ -62,14 +63,38 @@ foreach($BoostVersion in $BoostVersionsToInstall)
     Validate-BoostVersion -BoostRootPath $BoostRootDirectory -BoostRelease $BoostVersion
     $BoostVersionTag = "BOOST_ROOT_{0}" -f $BoostVersion.Replace('.', '_')
 
+    if ([Version]$BoostVersion -ge "1.70.0")
+    {
+        $BoostToolsetName = "msvc-14.2"
+    }
+    else
+    {
+        $BoostToolsetName = "msvc-14.1"
+    }
```

Comment:
I am not sure that this condition is true for Windows-2016 image. On this image, both 1.69 and 1.72 are `msvc-14.1`.
I believe the better approach will be looking into `https://github.com/actions/virtual-environments/blob/master/images/win/toolcache-2019.json` and parse toolset from it.

Or we can just complicate condition for now and improve it during switching to the GitHub releases for tool-cache